oak/proto/application.proto

```diff
@@ -94,6 +94,11 @@ message GrpcClientConfiguration {
   // The endpoint address of the external gRPC service.
   // `address` is represented as an "ip_address:tcp_port" string.
   string address = 1;
+  // Domain name of the external gRPC service.
+  string domain_name = 2;
+  // Loaded PEM encoded X.509 TLS root certificate file used to authenticate an external gRPC
```

Comment:
Is this the only cert that is used for authentication? Are any other root CAs installed on the machine ignored?

Reply:
IIUC they are ignored and will only be used if we enable the following feature:
```
feature = "tls-roots"
```
https://github.com/hyperium/tonic/blob/04a8c0c82a4007f48c3bf3539a3f2312746fedd1/tonic/src/transport/service/tls.rs#L59-L62